index.html

```diff
@@ -5,8 +5,33 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="style.css">
+  <script src="https://kit.fontawesome.com/e2597287a1.js" crossorigin="anonymous"></script>
   <title>Portfolio</title>
 </head>
 <body>
+  <div class="app-cont">
```

Comment:
- [ ] [OPTIONAL] it would be better if you change this `<div>` to `<main>` because it's the main element that has all other elements inside in it